src/ios/TLRKFeedback.m

```diff
@@ -7,11 +7,16 @@
 
 #import "TLRKFeedback.h"
 #import "CDVDevice.h"
-#import <TelerikAppFeedback/AppFeedback.h>
+#import <TelerikAppFeedback/TelerikAppFeedback.h>
 #import <objc/message.h>
 
 @implementation TLRKFeedback
 
+NSString * const feedbackSentAlertTitleSettingName = @"feedbackSentAlertTitle";
```

Comment:
Although not significantly important, I'd suggest to use some prefix for all constant declarations (e.g. `TLRK`) or starting with capital letter at least. Another good choice is to have the prefix `k` e.g. `kFeedbackSentAlertTitleSettingName`.